src/app/students/GroupMembers.js

```diff
@@ -1,37 +1,37 @@
+import React, {useEffect, useState} from "react";
 import {withRouter} from "react-router";
+import {useParams} from "react-router-dom";
+import {AiOutlineMail} from "react-icons/ai";
+import {studentService} from "../../services/services";
 import {GroupInPageNavigationBar} from "./GroupInPageNavigationBar";
 import {ItemListPage} from "../commons/ItemListPage/ItemListPage";
 import {CreateButton} from "../commons/buttons/CreateButton";
 import FakeLink from "../commons/FakeLink";
-import React, {useEffect, useState} from "react";
 import {ThreeDotsButton} from "../commons/buttons/ThreeDotsButton";
-import {studentService} from "../../services/services";
 import {Spinner} from "../commons/Spinner";
-import {AiOutlineMail} from "react-icons/ai";
 import {RemoveConfirmationModal} from "../commons/modals/RemoveConfirmationModal";
 import {removeIf} from "../../utils/array";
 import {TextareaModal} from "../commons/modals/TextareaModal";
+import {useGroupContext} from "./GroupContext";
 
 const GroupMembers = withRouter(({history, match}) => {
-    const currentPathName = history.location.pathname;
-    const [group, setGroup] = useState(undefined);
+    const currentURL = history.location.pathname;
```

Comment:
use `const {url: currentURL} = useRouteMatch()`, so you can replace the `withRouter(...)`(the `)` is at line 129) with `({match})`. This will improve the readability since history is not a props.